stochss/handlers/util/ensemble_simulation.py

```diff
@@ -59,9 +59,12 @@ def __init__(self, path, preview=False):
 
 
     def __get_run_settings(self):
-        solver_map = {"SSA":SSACSolver(model=self.g_model), "Tau-Leaping":TauLeapingSolver,
+        solver_map = {"SSA":SSACSolver, "Tau-Leaping":TauLeapingSolver,
```

Comment:
I think the new C++ solvers (`TauLeapingCSolver` and `ODECsolver`) are available in GillesPy2 v1.5.10, do we want to include those?

Reply:
That will be in the 2.3.1 release

Reply:
Ah, right! I see the Issue now.